src/components/Guilds/ActionsBuilder/CallDetails/index.tsx

```diff
@@ -58,6 +62,35 @@ const CallDetails: React.FC<ActionViewProps> = ({ decodedCall }) => {
 
   return (
     <>
+      {!!approveSpendTokens && (

```

Comment:
This part should follow the exact same format as the ABI is in, so `approve(address, uint256)` where the first parameter is spender and the second is amount 

(I know we then wouldn't show the token being approved so made an issue to add this to the design - fine to leave it for now)

Reply:
Updated with approval amount+token above and approval (spender, amount) below. 
![Screen Shot 2022-05-02 at 15 22 44](https://user-images.githubusercontent.com/25844967/166303833-554bcf31-0fdd-4d1d-b054-627cad7c53fc.png)